From 6b782dadcdd5bd7e15f382b0ace56170525d7145 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Fri, 8 Jun 2018 23:12:42 +0000
Subject: [PATCH] OS-7007 qede_ring_tx is too noisy Reviewed by: Jason King
 <jason.king@joyent.com> Approved by: John Levon <john.levon@joyent.com>

---
 usr/src/uts/common/io/qede/qede_fp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/usr/src/uts/common/io/qede/qede_fp.c b/usr/src/uts/common/io/qede/qede_fp.c
index 7d415cbb2a..8018dac365 100644
--- a/usr/src/uts/common/io/qede/qede_fp.c
+++ b/usr/src/uts/common/io/qede/qede_fp.c
@@ -1772,8 +1772,6 @@ qede_ring_tx(void *arg, mblk_t *mp)
 	}
 
 	if (!qede->params.link_state){
-		qede_print_err("!%s(%d): Link !up for xmit",
-		    __func__, qede->instance);
 		goto exit;
 	}
 
-- 
2.21.0

